### PR TITLE
Release v1.5.0

### PR DESCRIPTION
**Pull request by thundron**
### Misc

- Reworked flashing logic with [etcher-sdk](https://github.com/balena-io-modules/etcher-sdk)
- Upgrade to Electron v3.
- Upgrade to NPM 6.7.0
- Fix incorrect drives list on Linux
- Changed “Drive Contains Image” to “Drive Mountpoint Contains Image”
- Removed etcher-cli

### CODE REVIEW

**Comment by lurch** [comment]
Does the updated Electron version now include the Windows native drive locking that I remember being talked about a few years ago (that it was hoped would fix a whole swathe of Windows-specific issues) ? :crossed_fingers: 

Are there any user-visible benefits (yet?) to Etcher now using the etcher-sdk, that it might be worth explaining about in the release notes? :slightly_smiling_face: 

Perhaps the removal of the CLI #2630 should be mentioned too?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+## v1.5.0 - 2019-02-16
+
+### Misc
+
+- Reworked flashing logic with etcher-sdk
+- Upgrade to Electron v3.
+- Upgrade to NPM 6.7.0
+- Fix incorrect drives list on Linux
+- Changed “Drive Contains Image” to “Drive Mountpoint Contains Image”
+- Deprecated etcher-cli
```
Comment (zvin) [inline CHANGELOG.md]: removed, not deprecated

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.5.0 - 2019-02-16
+
+### Misc
+
+- Reworked flashing logic with etcher-sdk
+- Upgrade to Electron v3.
+- Upgrade to NPM 6.7.0
```
Comment (zvin) [inline CHANGELOG.md]: Not sure it's worth mentioning npm as it only affects us.

Comment (thundron) [reply CHANGELOG.md]: Us, and anyone that decides to contribute to Etcher, so yes it's worth imho

**Comment by lurch** [comment]
@thundron Gentle reminder that while the website has been updated to offer v1.5.0, https://deb.etcher.io still only has v1.4.9 available :wink:

**Comment by thundron** [comment]
@lurch We're trying to keep that clean, putting there only stable releases

**Comment by lurch** [comment]
Ahh, I didn't know v1.5.0 wasn't considered stable yet.

**Comment by der-eismann** [comment]
What exactly is considered stable by you? And are you tagging a new release for every commit now?

**Pull request merged**